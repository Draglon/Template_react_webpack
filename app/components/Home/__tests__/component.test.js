import React from 'react';

import HomeComponent from '../component';

describe('<HomeComponent />', () => {
  const props = { sessionId: '' };
  const component = shallow(<HomeComponent {...props} />);

  describe('sessionId: false', () => {
    it('should match snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });

  describe('sessionId: true', () => {
    it('should match snapshot', () => {
      component.setProps({ sessionId: 'some sessionId' });
      expect(component).toMatchSnapshot();
    });
  });
});
