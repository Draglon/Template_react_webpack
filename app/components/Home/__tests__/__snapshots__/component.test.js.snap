// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeComponent /> sessionId: false should match snapshot 1`] = `
<Fragment>
  <Connect(LoginFormContainer) />
</Fragment>
`;

exports[`<HomeComponent /> sessionId: true should match snapshot 1`] = `
<Fragment>
  <Connect(DashboardComponent) />
</Fragment>
`;
